ddlibrary/ddlibrary
# Fix some issues

On this PR I have fixed some issues and add Recaptcha v3.

1. Remove facebook sdk
2. Fix GlossaryListTest's namespace 
3. Create .env.example
4. Add subscriber to registration form and add the text on language files
5. Add reCaptcha V3
6. Create RecaptchaRule for validation base on [documentation](https://developers.google.com/recaptcha/docs/verify) 
7. Fix ddlibrary failed issues

Note: please create reCaptcha site and secret keys from this [link](https://www.google.com/recaptcha/admin/create)

> Please run `php artisan migrate` , ` composer install` and `npm run production`
